torch/csrc/autograd/functions/basic_ops.h

```diff
@@ -25,12 +25,17 @@ struct Error : public Function {
 
 // Identity in forward, Error in backward. Used to implement @once_differentiable
 struct DelayedError : public Function {
-  DelayedError(std::string msg)
-    : msg(std::move(msg)) {};
+  DelayedError(std::string msg, int num_inputs)
+    : msg(std::move(msg)) {
+      for (int i = 0; i < num_inputs; i++)
+        input_metadata_.emplace_back();
```

Comment:
I think the recommended way to do this is `add_input_metadata(Function::undefined_input())`

Reply:
Sure, will modify. I thought it was the same thing, so I did it this way.